### PR TITLE
Rebuild storage service DB

### PR DESCRIPTION
**Pull request by Hwesta**
Add ability to rebuild the SS DB by walking a filesystem.

refs #7250

### CODE REVIEW

File: tools/rebuild-storage-service-db [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+def main(path, ss_url, space, location, pipeline):
+    api_version = '2'
+
+    # Check if Space exists
+    url = 'http://{}/api/v{}/space/{}/'.format(ss_url, api_version, space)
+    response = requests.get(url, verify=False)
+    if response.status_code == 404:
+        # Create new local space
+        print('TODO create local space')
```
Comment (mistydemeo) [inline tools/rebuild-storage-service-db]: Is this a TODO before this gets merged?

File: tools/rebuild-storage-service-db [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        # Create new local pipeline
+        print('TODO create local pipeline')
+        return 2
+    elif response.status_code == 200:
+        print('Pipeline', pipeline, 'exists')
+        pipeline_info = response.json()
+
+    # Walk directory
+    print('Searching', path, 'for AIPS...')
+    aip_name_regex = r'^(?P<name>\w*)-(?P<uuid>\w{8}-\w{4}-4\w{3}-\w{4}-\w{12}).(?P<extension>7z|tar.bz2)$'
```
Comment (mistydemeo) [inline tools/rebuild-storage-service-db]: Will `\w` match all characters that can be in an AIP name? I think `-` is also valid.

File: tools/rebuild-storage-service-db [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                    'status': 'UPLOADED',
+                    'uuid': match.group('uuid'),
+                }
+                print('payload', payload)
+                url = 'http://{}/api/v{}/file/'.format(ss_url, api_version)
+                print('url', url)
+                response = requests.post(url, verify=False, data=json.dumps(payload), headers=headers)
+                print('response', response)
+                print(response.text)
+        # Search dirs for UC AIPs
```
Comment (mistydemeo) [inline tools/rebuild-storage-service-db]: This still TODO?

File: tools/rebuild-storage-service-db [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+                    'origin_pipeline': pipeline_info['resource_uri'],
+                    'package_type': 'AIP',
+                    'size': str(os.path.getsize(abs_path)),
+                    'status': 'UPLOADED',
+                    'uuid': match.group('uuid'),
+                }
+                print('payload', payload)
+                url = 'http://{}/api/v{}/file/'.format(ss_url, api_version)
+                print('url', url)
+                response = requests.post(url, verify=False, data=json.dumps(payload), headers=headers)
```
Comment (mistydemeo) [inline tools/rebuild-storage-service-db]: We explicitly set `http` above; is `verify=False` doing anything? Is the SS redirecting to an invalid cert in some scenarios?

**Comment by sevein** [comment]
I'm closing as this is not being actively developed.
I've filed a feature request: https://github.com/archivematica/Issues/issues/229.

**Pull request closed without merging**